### DISCUSSION TITLE
折叠动态突然失效了

### DISCUSSION THREAD

**Discussion by KQQQQOVO** [the1812/Bilibili-Evolved#4190] [综合讨论 / General]
鼠标放到动态时没有![Snipaste_2023-05-09_11-56-06](https://user-images.githubusercontent.com/75309934/236990205-371f7bd1-c73f-4c6e-b08e-70c8e450a4b3.png)果了，变成了原版的了，请问怎么恢复到简化后的效果？收藏夹也得点开了

**Comment by Geeyun-JY3** [comment]
呃你要不看看你在说什么？
你是要`自定义顶栏`？

**Comment by KQQQQOVO** [comment]
哦，我的我的 不知道怎么关了 sry




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年5月9日(星期二) 下午5:19
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [the1812/Bilibili-Evolved] 折叠动态突然失效了 (Discussion #4190)





 
呃你要不看看你在说什么？
 你是要自定义顶栏？
